src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -6349,6 +6351,122 @@ components:
         exclusive:
           $ref: '#/components/schemas/ShortIdProperty'
 
+    ##############################################################################
+    ## SNAPSHOT BUILDER API STANDARD MODELS
+    ##############################################################################
+    SnapshotBuilderConfig:
+      type: object
+      description: >
+        The configuration describing how to select criteria for issuing a snapshot builder request
```

Comment:
Do we want to put a note here that this model is subject to change?

Reply:
I agree, I think all the snapshot builder APIs and object should have ⚠️ or similar in their descriptions.

Reply:
I added the ⚠️, as well as a bit at the header for the model section that all the models are subject to change.